templates/repo/release/list.tmpl

```diff
@@ -77,8 +77,9 @@
 							<div class="markup desc">
 								{{Str2html .Note}}
 							</div>
-							<details class="download gt-border-secondary-top gt-mt-4 gt-pt-4" {{if eq $idx 0}}open{{end}}>
-								<summary class="gt-mb-4">
+							<div class="divider"></div>
+							<details class="download" {{if eq $idx 0}}open{{end}}>
```

Comment:
offtopic, but `download` sounds strange.
Wouldn't that semantically be `release-assets`?

Reply:
There are already many concepts: `download`, `asset`, `attachment`, etc. Keeping the old `download` doesn't bother me.